Cargo.toml

```diff
@@ -183,11 +183,8 @@ features = [
   "Win32_Foundation",
   "Win32_Storage_FileSystem",
   "Win32_System_Console",
-  "Win32_System_IO",
   "Win32_System_Threading",
   "Win32_System_JobObjects",
-  "Win32_Security",
```

Comment:
This should't be removed, no? At least `cargo-credential-wincred` needs both `Win32_Security_Credentials` and `Win32_Security` AFAICT. Maybe move this to `cargo-credential-wincred`'s Cargo.toml?